src/pandroid/app/src/main/java/com/panda3ds/pandroid/view/PandaGlSurfaceView.java

```diff
@@ -0,0 +1,46 @@
+package com.panda3ds.pandroid.view;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.opengl.GLSurfaceView;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import com.panda3ds.pandroid.math.Vector2;
+import com.panda3ds.pandroid.utils.Constants;
+import com.panda3ds.pandroid.view.controller.TouchEvent;
+import com.panda3ds.pandroid.view.controller.nodes.TouchScreenNodeImpl;
+import com.panda3ds.pandroid.view.renderer.ConsoleRenderer;
+
+public class PandaGlSurfaceView extends GLSurfaceView implements TouchScreenNodeImpl {
+	final PandaGlRenderer renderer;
+	private int size_width;
```

Comment:
Inconsistent variable naming with the other Java variables that use camelCase. Should be called viewWidth/viewHeight (size_width is also redundant since width implies a size but the name doesn't tell you what size)